src/08_comprehensions.py

```diff
@@ -36,6 +34,8 @@
 x = input("Enter comma-separated numbers: ").split(',')
 
 # What do you need between the square brackets to make it work?
-y = []
-
+y = [z for z in x if int(z) %2 == 0]
```

Comment:
Great work on the list comprehensions! Fortunately we took those notes together, so you won't forget all the things that we discussed/practice regarding list comprehensions! :)
![Screen Shot 2019-12-09 at 10 05 52 AM](https://user-images.githubusercontent.com/16110931/70446793-a331e380-1a6b-11ea-9d0c-cebccfee61a9.png)